### PR TITLE
Add alert support

### PR DESCRIPTION
**Pull request by debendraoli**
This pr add supports for alerting to discord webhooks and telegram bots when wallet balance is below the threshold.

### CODE REVIEW

File: wallets.json [hunk 1]
```diff
@@ -29,6 +31,7 @@
             "rpc": "https://ctz.solidwallet.io/api/v3/icon_dex",
             "type": "icon",
             "decimals": 18,
+            "threshold": "0.1",
```
Comment (izyak) [inline wallets.json]: ICX gets used easily, especially for BTP, so lets have a larger threshold.

200 ICX should be alright

File: wallets.json [hunk 2]
```diff
@@ -44,6 +47,7 @@
             "rpc": "https://api.mainnet.archway.io",
             "type": "cosmos",
             "decimals": 18,
+            "threshold": "0.1",
```
Comment (izyak) [inline wallets.json]: same, ssth like 50 arch

File: wallets.json [hunk 3]
```diff
@@ -56,6 +60,7 @@
             "rpc": "https://api.novel.remedy.tm.p2p.org",
             "type": "cosmos",
             "decimals": 6,
+            "threshold": "0.1",
```
Comment (izyak) [inline wallets.json]: maybe 20 here for now, till there is active ibc between icon and neutron

File: wallets.json (hunk 1 above)
Comment (debendraoli) [reply wallets.json]: Sure, it was just an example for quick test

**Pull request merged**